packages/babel-helper-create-class-features-plugin/test/fixtures/typescript/constructor-order-super/output.js

```diff
@@ -0,0 +1,24 @@
+class Foo {}
+class Foo2 extends Foo {
+  constructor(incoming) {}
+}
+class Foo3 extends Foo {
+  constructor(incoming) {
+    super();
+    this.incoming = incoming;
```

Comment:
Unrelated, but in Babel 8 we should emit this based on the `setPublicClassFields` assumption.